### PR TITLE
a new folder for styling

### PR DESCRIPTION
**Pull request by baidi-mannan**
A new folder for styles added and the name of the styles is changed.
This is for Issue #7

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,9 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="stylesheets/index.css">
     <link href="https://fonts.googleapis.com/css?family=Fredericka+the+Great&display=swap" rel="stylesheet">
-    <title>New Tab</title>
+    <title>DarkSite</title>
```
Comment (sudo-rgorai) [inline index.html]: I don't think it's a good idea to change the title of the page. A new tab should **feel** like a new tab. Seeing the `New Tab` title preserves the familiarity of the page. I suggest you to restore this.

File: user.html [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="stylesheets/user.css">
     <title>New Tab</title>
 </head>
 <body>
     <img src="images/favicon-128x128.png" class="logo">
     <h1>What's your name?</h1>
     <form action="index.html">
-        <input type="text" name="username" id="usr-input">
+        <input type="text" name="username" id="usr-input" placeholder="Type Here...">
```
Comment (sudo-rgorai) [inline user.html]: Again, this is not directly related to the issue. A PR should only do what it claims to do. For any additional changes, you are free to add a new issue and send a specific PR related to that. This change is fine, however and you may keep this if you want.

File: index.html (hunk 1 above)
Comment (baidi-mannan) [reply index.html]: Okay will do that!!

File: user.html (hunk 2 above)
Comment (baidi-mannan) [reply user.html]: Okay, Sorry for that..

Comment (sudo-rgorai) [reply user.html]: No problem :smile:

**Comment by sudo-rgorai** [comment]
LGTM :+1:
